cmake/toolchain/support/arduino-support.cmake

```diff
@@ -196,7 +200,7 @@ function(finalize_arduino_executable)
         string(REPLACE "<TARGET_NAME>" "$<TARGET_FILE_NAME:${FPRIME_CURRENT_MODULE}>" COMMAND_WITH_INPUT "${COMMAND_WITH_INPUT}")
         string(REPLACE "<TARGET_DIRECTORY>" "$<TARGET_FILE_DIR:${FPRIME_CURRENT_MODULE}>" COMMAND_WITH_INPUT "${COMMAND_WITH_INPUT}")
         string(REPLACE " " ";" COMMAND_WITH_INPUT "${COMMAND_WITH_INPUT}")
-        list(APPEND COMMAND_SET_ARGUMENTS COMMAND ${COMMAND_WITH_INPUT} || true)
+        list(APPEND COMMAND_SET_ARGUMENTS COMMAND ${COMMAND_WITH_INPUT})
```

Comment:
I think I remember the rationale for putting `|| true` here.

Some toolchains like the `featherM0` try to run `xdg-open`. Why? Not sure. But this error appears without the `|| true`:
```
xdg-open: unexpected argument 'no'
Try 'xdg-open --help' for more information.
gmake[2]: *** [LedBlinker/CMakeFiles/LedBlinker.dir/build.make:346: bin/featherM0/LedBlinker] Error 1
gmake[2]: *** Deleting file 'bin/featherM0/LedBlinker'
gmake[1]: *** [CMakeFiles/Makefile2:7406: LedBlinker/CMakeFiles/LedBlinker.dir/all] Error 2
gmake: *** [Makefile:136: all] Error 2
[ERROR] CMake erred with return code 2
```
I vaguely remember running into this while we were working with the Feather M0 last year.

Reply:
Also unable to build for RP2040 without `|| true` for `rp2040@4.0.0` and newer. But for a different reason.

```
ERROR: filename '/home/ethancheez/Documents/Git/fprime-projects/fprime-tutorial-arduino-blinker/build-fprime-automatic-rpipico/bin/rpipico/LedBlinker' does not have a recognized file type (extension)
gmake[2]: *** [LedBlinker/CMakeFiles/LedBlinker.dir/build.make:349: bin/rpipico/LedBlinker] Error 157
gmake[2]: *** Deleting file 'bin/rpipico/LedBlinker'
gmake[1]: *** [CMakeFiles/Makefile2:7406: LedBlinker/CMakeFiles/LedBlinker.dir/all] Error 2
gmake: *** [Makefile:136: all] Error 2
[ERROR] CMake erred with return code 2
```

Reply:
I made it better to apply to all arduino products.